hypothesis-python/tests/nocover/test_from_type_recipe.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Most of this work is copyright (C) 2013-2019 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import absolute_import, division, print_function
+
+import datetime
+import decimal
+import fractions
+import numbers
+import uuid
+
+import hypothesis.strategies as st
+from hypothesis import given
+
+
+def everything_except(excluded_types):
+    """Recipe copied from the docstring of ``from_type``"""
+    return (
+        st.from_type(type)
+        .flatmap(st.from_type)
+        .filter(lambda x: not isinstance(x, excluded_types))
+    )
+
+
+many_types = [
```

Comment:
Let's steal the idea from #1974: `sorted([x for x in _global_type_lookup if x.__module__ != "typing"], key=str)`